### ISSUE TITLE
DVD's won't rip

### ISSUE THREAD

**Issue by stre1026** [automatic-ripping-machine/automatic-ripping-machine#112]
I've been using ARM for a week to rip Blu-Rays and that is working perfectly. I Just tried to rip a few DVD's today and the system won't do anything with a DVD. It just grabs the DVD name and then ends because it says it can't find a file to transcode. I am able to rip a DVD manually with HandbrakeCLI so I'm pretty sure that's working OK.

What can I check?

Here's one of the logs:

Starting Identify Script...
Deleting 0 old log files:
DEVLINKS=/dev/disk/by-path/pci-0000:00:14.1-ata-2 /dev/disk/by-id/ata-HL-DT-ST_BDDVDRW_UH12NS30_K96F2P84706 /dev/dvd /dev/disk/by-label/HOME2 /dev/disk/by-uuid/1999-08-03-13-11-14-00 /dev/cdrw /dev/dvdrw /dev/cdrom
DEVNAME=/dev/sr0
DEVPATH=/devices/pci0000:00/0000:00:14.1/ata2/host1/target1:0:1/1:0:1:0/block/sr0
DEVTYPE=disk
ID_ATA=1
ID_ATA_SATA=1
ID_ATA_SATA_SIGNAL_RATE_GEN1=1
ID_BUS=ata
ID_CDROM=1
ID_CDROM_BD=1
ID_CDROM_CD=1
ID_CDROM_CD_R=1
ID_CDROM_CD_RW=1
ID_CDROM_DVD=1
ID_CDROM_DVD_PLUS_R=1
ID_CDROM_DVD_PLUS_RW=1
ID_CDROM_DVD_PLUS_R_DL=1
ID_CDROM_DVD_R=1
ID_CDROM_DVD_RAM=1
ID_CDROM_DVD_RW=1
ID_CDROM_MEDIA=1
ID_CDROM_MEDIA_DVD=1
ID_CDROM_MEDIA_SESSION_COUNT=1
ID_CDROM_MEDIA_STATE=complete
ID_CDROM_MEDIA_TRACK_COUNT=1
ID_CDROM_MEDIA_TRACK_COUNT_DATA=1
ID_FOR_SEAT=block-pci-0000_00_14_1-ata-2
ID_FS_LABEL=HOME2
ID_FS_LABEL_ENC=HOME2
ID_FS_TYPE=udf
ID_FS_USAGE=filesystem
ID_FS_UUID=1999-08-03-13-11-14-00
ID_FS_UUID_ENC=1999-08-03-13-11-14-00
ID_MODEL=HL-DT-ST_BDDVDRW_UH12NS30
ID_MODEL_ENC=HL-DT-ST\x20BDDVDRW\x20UH12NS30\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20
ID_PATH=pci-0000:00:14.1-ata-2
ID_PATH_TAG=pci-0000_00_14_1-ata-2
ID_REVISION=1.03
ID_SERIAL=HL-DT-ST_BDDVDRW_UH12NS30_K96F2P84706
ID_SERIAL_SHORT=K96F2P84706
ID_TYPE=cd
MAJOR=11
MINOR=0
SUBSYSTEM=block
TAGS=:uaccess:seat:systemd:
USEC_INITIALIZED=3158497
identified udf
found HOME2 on /dev/sr0
identified udf as video
Obtained Title Home Alone 2 - Lost in New York (2015)
VIDEO_TYPE is movie and NEW_YEAR is 1992
Year mismatch found.  New video title is Home Alone 2 - Lost in New York (1992)
got to here
HAS_NICE_TITLE is true
video title is now Home Alone 2 - Lost in New York (1992)
video type is movie
Starting video_rip.sh
Video Title is Home Alone 2 - Lost in New York (1992)
Ripping video HOME2 from /dev/sr0
Media is DVD and Main Feature parameter in config file is true.  Bypassing MakeMKV.
DEST is /home/arm/movies/output/raw/Home Alone 2 - Lost in New York (1992)_20180120_183951
STAT: HOME2 ripped in 0 hours, 0 minutes and 0 seconds.
/opt/arm/video_transcode.sh "/home/arm/movies/output/raw/Home Alone 2 - Lost in New York (1992)_20180120_183951" "Home Alone 2 - Lost in New York (1992)" "true" "movie" "20180120_183951" "/opt/arm/logs/HOME2.log"
Home Alone 2 - Lost in New York (1992) sent to transcoding queue...
Start video transcoding script
transcoding with a nice title
directory already exists... adding timestamp
Transcoding all files.
$VIDEO_TYPE is movie, $MAINFEATURE is true, $HAS_NICE_TITLE is true, $EMBY_SUBFOLDERS is false
Moving a single file.
Checing for existing file...
No file found.  Moving "/home/arm/movies/output/unidentified/Home Alone 2 - Lost in New York (1992)_20180120_183951/Home Alone 2 - Lost in New York (1992).mkv to /home/arm/movies/output/final/Home Alone 2 - Lost in New York (1992).mkv"
Emby Refresh False.  Skipping library scan
STAT: Home Alone 2 - Lost in New York (1992) transcoded in 0 hours, 0 minutes and 0 seconds.
Pushbullet notifications not enabled
IFTTT notifications not enabled
Pusover notifications not enabled
Sending EMail notification
EMail notification sent

**Comment by muckngrind4** [comment]
Hi @stre1026, can you post your config file also?  (Make sure to sanitize it of any keys/email/etc.).  From what I can tell, it's a DVD and you have MAINFEATURE as true, so it should be trying to rip just the main title.  However the "Transcoding all files." line in the log means it went a different path for some reason.  If I can see your config file it will help.

**Comment by stre1026** [comment]
Woops! Looks like my account was blocked due to some automated process thinking I was spamming the site. Got that taken care of. Anyone have any ideas? Looks like my config file copied a little funny.

**Comment by muckngrind4** [comment]
Just saw this.  Sorry.  I'll check it out tomorrow.

**Comment by muckngrind4** [comment]
Hi @stre1026,
Any chance the config file changed in the middle of the processing?
Does it work if you make mainfeature false?
 I'm especially curious on the last point.

**Comment by audioeng** [comment]
I don’t use the main feature flag and it works great for dvd and Blu-ray
for me.

**Comment by stre1026** [comment]
Hey @muckngrind4,

The config definitely didn't change during processing. It also didn't matter whether mainfeature was false or not. However, what I did do because @audioeng said his is working fine replaced my config with a fresh copy and all I changed was the paths to my media directories in the new config. I left everything as default for now. This worked fine and I was able to rip a DVD. What I will do next is start adding things back in like my MakeMKV key, API key for OMDB and E-mail config.
